src/main.h

```diff
@@ -1303,6 +1303,7 @@ class CBlockIndex
     // Gridcoin (17.6.2017 Brod) Add dpor related back-pointers
     CBlockIndex* ppCpid;  // previous block by same cpid
     CBlockIndex* ppSuper; // previous superblock
+    CSuperblock* pDesSuper; // deserialized superblock data

```

Comment:
Yeah I've seen that but hopefully we can implement a block index "appender" which tosses out old blocks. The loader might become a littler slower due to this but we'd need it at runtime anyway. Anyway, a slow loading chain is better than a high requiring one, and the loader should be simpler to fix.